Pre go-live cleanup of ff docs (#215)
harness/developer-hub: docs/feature-flags/5-subscribe-ff/1-subscriptions-overview.md

Using online subscriptions, you can choose whether to be billed yearly or monthly, and can select the exact number of licenses you need for your developers and for monthly active users (MAUs). It’s as easy as signing in to the Harness Platform, choosing your [subscription plan](3-available-plans-for-feature-flags.md), and entering your billing details.  

:::note

> Currently online subscriptions are available only to self-serve Feature Flags customers. For enterprise buyers or for subscribing to other modules, contact our Sales team.

Reviewer: ```suggestion
Currently online subscriptions are available only to self-serve customers. If you are an enterprise buyer or want to subscribe to other modules, contact the Harness Sales team.
```
